<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BDC6-DX-5F1HXH2" ignored="0" name="Test" tests="15" failures="5" timestamp="17 Sep 2019 10:07:12 GMT" time="0.119" errors="0">
  <testcase name="testmethod1" time="0.009" classname="day_1.testng_demo3"/>
  <testcase name="testmethod2" time="0.002" classname="day_1.testng_demo3"/>
  <testcase name="testmethod3" time="0.002" classname="day_1.testng_demo3">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at day_1.testng_demo3.testmethod3(testng_demo3.java:18)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testmethod3 -->
  <testcase name="testmethod4" time="0.002" classname="day_1.testng_demo3"/>
  <testcase name="testmethod5" time="0.002" classname="day_1.testng_demo3">
    <failure type="java.lang.AssertionError" message="expected [30] but found [20]">
      <![CDATA[java.lang.AssertionError: expected [30] but found [20]
at day_1.testng_demo3.testmethod5(testng_demo3.java:28)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testmethod5 -->
  <testcase name="testmethod6" time="0.002" classname="day_1.testng_demo3">
    <failure type="java.lang.AssertionError" message="expected [Hi] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hi] but found [Hello]
at day_1.testng_demo3.testmethod6(testng_demo3.java:34)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testmethod6 -->
  <testcase name="testmethod7" time="0.001" classname="day_1.testng_demo3"/>
  <testcase name="testmethod8" time="0.001" classname="day_1.testng_demo3">
    <failure type="java.lang.AssertionError" message="expected [null] but found [java.lang.Object@2eb77042]">
      <![CDATA[java.lang.AssertionError: expected [null] but found [java.lang.Object@2eb77042]
at day_1.testng_demo3.testmethod8(testng_demo3.java:44)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testmethod8 -->
  <testcase name="testmethod9" time="0.008" classname="day_1.testng_demo3">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [11] but found [10],
	expected [hi] but found [hello]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [11] but found [10],
expected [hi] but found [hello]
at day_1.testng_demo3.testmethod9(testng_demo3.java:55)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testmethod9 -->
  <testcase name="f" time="0.002" classname="day_1.testng_demo"/>
  <testcase name="f1" time="0.001" classname="day_1.testng_demo"/>
  <testcase name="Register" time="0.002" classname="day_1.demo2"/>
  <testcase name="Login" time="0.005" classname="day_1.demo2"/>
  <testcase name="HomePage" time="0.002" classname="day_1.demo2"/>
  <testcase name="Logout" time="0.001" classname="day_1.demo2"/>
</testsuite> <!-- Test -->
